[2025-06-18 15:08:10,072] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: first_dag.second_task 2024-09-19T00:00:00+00:00 [queued]>
[2025-06-18 15:08:10,179] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: first_dag.second_task 2024-09-19T00:00:00+00:00 [queued]>
[2025-06-18 15:08:10,182] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2025-06-18 15:08:10,182] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2025-06-18 15:08:10,182] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2025-06-18 15:08:10,332] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2024-09-19T00:00:00+00:00
[2025-06-18 15:08:10,346] {standard_task_runner.py:52} INFO - Started process 14481 to run task
[2025-06-18 15:08:10,363] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'first_dag', 'second_task', '2024-09-19T00:00:00+00:00', '--job-id', '13104', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpel36a95m', '--error-file', '/tmp/tmpuu8q56m8']
[2025-06-18 15:08:10,371] {standard_task_runner.py:77} INFO - Job 13104: Subtask second_task
[2025-06-18 15:08:10,726] {logging_mixin.py:104} INFO - Running <TaskInstance: first_dag.second_task 2024-09-19T00:00:00+00:00 [running]> on host 500dc4ac3a72
[2025-06-18 15:08:10,994] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=my_name
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2024-09-19T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-09-19T00:00:00+00:00
[2025-06-18 15:08:11,001] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2025-06-18 15:08:11,002] {bash.py:158} INFO - Running command: echo "this is sencond task "
[2025-06-18 15:08:11,029] {bash.py:169} INFO - Output:
[2025-06-18 15:08:11,030] {bash.py:173} INFO - this is sencond task
[2025-06-18 15:08:11,030] {bash.py:177} INFO - Command exited with return code 0
[2025-06-18 15:08:11,174] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=second_task, execution_date=20240919T000000, start_date=20250618T150810, end_date=20250618T150811
[2025-06-18 15:08:11,300] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-18 15:08:11,358] {local_task_job.py:146} INFO - Task exited with return code 0
